tutorials/NWB Tutorials/write_to_NWB.m

```diff
@@ -0,0 +1,83 @@
+%% WRITE TO NWB TUTORIAL
+% This tutorial covers two approaches for writing EXTRACT output data to
+% file:
+%   1. Writing data to a new NWB file
+%   2. Writing data to an existing NWB file
+%% *NWB Extension Setup*
```

Comment:
Can you also include similar instructions for downloading and installing MatNWB? I don't think we can assume that users of this tutorial have already done the other tutorial. Alternatively, we could refactor out the MatNWB installation instructions into its own tutorial and refer to it. Either way is fine with me.